packages/client/webpack.config.ts

```diff
@@ -20,7 +20,7 @@ const config: Configuration = {
   },
 
   output: {
-    publicPath: '/'
+    publicPath: './'
```

Comment:
Why did we add this? This will probably break things in deeper routes. IE: if we go to `/login`.

Reply:
I was testing the client without "yarn dev" and it came up as a blank screen.
I added that current directory and it worked.

Then 10 minutes ago I saw you wrote a description to use "yarn dev", now it works.